lola.html

```diff
@@ -1010,6 +1187,26 @@ <h3>Following up a ‘Like’ of moved object</h3>
         Open Issue: should the Like activity then be rewritten with the new location? Can a client do this - is there an affordance to update a Like? 
       </p>
     </section>
+    <section class="informative">
+      <h2>Not In Scope</h2>
+      <p>
+        Interoperable formats for export/import are not in scope.  The collections defined to provide migration data can 
+        lead smoothly to an interoperable export/import format that can share much code with support for this specification; 
+        however many of the recommendations for this specification are tied to its use for re-homing content in a 
+        best-effort server-to-server move.  As an example, the destination server can skip over content it doesn't handle in an
+        account transfer, whereas export/import use cases could have a different approach due to many differences in
+        context.
+      </p>
+
+      <p>
+        Signatures are out of scope.
+      </p>
+      <p>
+        Nomadic identities (in particular, how to transfer the identity itself) are out of scope, though some recommendations
+        accounting for the existence of nomadic identities have been made.
```

Comment:
you might also want to add "witness services" that index/crawl/archive old publications à la internet archive. unless you hate the idea of such services existing so much you are afraid to manifest them 😅